documentation/source/coroutines.rst

```diff
@@ -202,3 +143,52 @@ type of generator function that can be iterated with ``async for``:
 
 More details on this type of generator can be found in :pep:`525`.
 
+
+.. _yield_syntax:
```

Comment:
```suggestion
.. _yield-syntax:
```
rst seems to prefer hyphenated targets

Reply:
Can you put a link to here in the `First` docs?

Reply:
Is a note at the bottom sufficient? I'm not sure where else would be a natural place to add that.

Reply:
Just add a link by the current "When using the old-style yield-based coroutines" note